webhook/webhook.py

```diff
@@ -26,24 +30,69 @@
 app.config['DEBUG'] = True
 
 @app.route('/')
-def hello_world():
+def status():
     return 'Airflow 2.5.3 `git pull` webhook @ ' + time.ctime()
 
 @app.route('/webhook', methods=['POST'])
 def handle_webhook():
-    payload = request.get_json()
+    # we only want to run this in production, so we check the environment variable
+    if OPERATING_ENVIRONMENT != "production":
+        app.logger.info("\nnot in production, skipping webhook\n")
+        return "Not in production, skipping webhook"
+    # get the client key from the request header
+    client_key = request.headers.get('X-Webhook-Key')
```

Comment:
related to my comment below, we should probably terminate the request right here if client get is falsey